src/utils/common/common.c

```diff
@@ -99,10 +99,7 @@ int ssnprintf(char *str, size_t sz, const char *format, ...) {
 
   va_end(ap);
 
-  if (ret < 0) {
-    return ret;
-  }
-  return (size_t)ret >= sz;
+  return ret;
```

Comment:
This merge is wrong too.

Reply:
This merge is https://github.com/collectd/collectd/issues/3232 aka https://github.com/collectd/collectd/commit/7a5f35e655e8ba8ccd08fd44bea3574dffa47a5e

Reply:
Ah yes you're right.